app/views/fact_values/index.html.erb

```diff
@@ -9,7 +9,7 @@
       <th><%= sort :name, :as => _("Name")  %></th>
       <th><%= sort :value, :as => s_("FactValue|Value") %></th>
       <th><%= _('Reported at') %></th>
-      <th></th>
+      <th><%= _('Distribution chart') %></th>
```

Comment:
Can you remove the word 'chart' from the button now that we have a title, so it's just the button with the glyphicon?